docs/generated.d.ts

```diff
@@ -762,6 +763,12 @@ export type NonFungibleTokenHolding = NonFungibleTokenHoldingWithTxId | NonFungi
  * Describes the minting of a Non-Fungible Token
  */
 export type NonFungibleTokenMint = NonFungibleTokenMintWithTxId | NonFungibleTokenMintWithTxMetadata;
+export type TransactionEvent1 =
```

Comment:
Can you remove the a 1 at the end here? I think you'll need to edit the TransactionEvent entity schema somewhere